utils/corpus.py

```diff
@@ -171,3 +181,61 @@ def tag_morphology(tag):
         k, v = p.split("=", 1)
         info[k] = v
     return info
+
+
+class FactRu(Corpus):
+
+    @staticmethod
+    def _resolve_data_path(data_path, download_if_not_exist):
+        data_path = data_path or os.path.join(nerus_const.SOURCES_DIR, nerus_const.FACTRU_DIR, 'master.zip')
+        print("FactRu data path: ", data_path)
+        if not os.path.exists(data_path):
+            if download_if_not_exist:
+                os.makedirs(os.path.dirname(data_path))
+
+                print("Download FactRu corpus to ", data_path)
+                with TqdmUpTo(unit='B', unit_scale=True, unit_divisor=1024, miniters=1,
+                              desc="FactRu corpus downloading") as tqdm_:
+                    try:
+                        urllib.request.urlretrieve(nerus_const.FACTRU_URL, data_path, reporthook=tqdm_.update_to)
+                        tqdm_.total = tqdm_.n
```

Comment:
drop download logic. add instruction to set/download the path to data.
example download in a notebook.